### ISSUE TITLE
"path": "/var/www/pimpmylog", "state": "absent"

### ISSUE THREAD

**Issue by danylevskyi** [hashbangcode/vlad#124]
That is the result of provision with default example.settings.yml.
```
TASK: [pimpmylog | ensure Pimpmylog directory permissions] ******************** 
failed: [vlad] => {"failed": true, "path": "/var/www/pimpmylog", "state": "absent"}
msg: recurse option requires state to be 'directory'

FATAL: all hosts have already failed -- aborting
```

**Comment by danbohea** [comment]
I just tested this using the latest dev version of Vlad and it produced no errors.

Please can you share more details about your setup?

- master or dev branch and what point in that branch?
- vagrant version
- ansible version
- etc

Cheers!

**Comment by danylevskyi** [comment]
Here are details:
Branch: master
Vagrant: 1.6.3
Ansible: 1.8.2
OS: Ubuntu 14.04

**Comment by danbohea** [comment]
> Branch: master

That could well be your problem right there. Dev is waaay ahead of master right now for a number of reasons and we're working hard to tag another release ASAP.

For the time being, please try the dev branch and be sure to update all variables set in your settings file to use `true` and `false` in place of `"y"` and `"n"` respectively.

Let me know how you get on!

**Comment by philipnorton42** [comment]
> msg: recurse option requires state to be 'directory'

I noticed this in the error output you posted before and thought that might be the problem. The Ansible documentation doesn't detail this issue (http://docs.ansible.com/file_module.html) but I thought it prudent to add it anyway. So I added the state parameter to this task in the dev branch. If this fixes your issue then we'll close this issue.

We are quite close to releasing a new version so hopefully this will be in the master branch soon. Thanks for your help :)
